src/enact/invocations.py

```diff
@@ -65,6 +65,16 @@ def set_from(self: ResourceT, other: Any):
 class NativeException(ExceptionResource):
   """A python exception wrapped as a resource."""
 
+  def __init__(self, type_name: str, str_representation: str, traceback: str):
+    self._type_name = type_name
+    self._str_representation = str_representation
+    self._traceback = traceback
+    super().__init__(traceback)
```

Comment:
You should probably pass this in as *args to the superclass, so it's compatible with normal exception functionality. I also don't think this is compatible with the resource interface, i.e., I think you won't be able to commit and checkout an exception of this type and keep all the detail.

Reply:
Done.